packages/ethereum/ethers-ethereum/src/index.ts

```diff
@@ -53,6 +58,46 @@ export class EthersWeb3ProviderEthereum implements EthereumProvider.Ethereum {
     }
   }
 
+  async sendTransaction(options: EthereumSendTransactionOptions): Promise<any> {
+    try {
+      let enrichedData = options.data || "0x"
```

Comment:
And what if options.data will be empty?
Then it will transfer everything to 0x0000 ?